### PR TITLE
VenueNames: header & CTA, highlight main venue

### PR DESCRIPTION
**Pull request by j0nas**
# VenueNames: header & CTA, highlight main venue

## Intent

Render new fields (and logic changes) for `venuesSection` in accordance with [Figma updates](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=6355%3A18309).

## Remaining TODOs before this PR is ready for review:

- [x] ~@kmelve According to the Figma sketches, it should be possible to configure each venues to link to an arbitrary target (hence the underlined venue names in the Figma sketches). Could you please advise on the best Studio schema structure to accomplish this?~ We are scoping linking out of the first iteration covered by this PR.
- [x] @oyvind-stenhaug Styling and markup changes

## Testing this PR

1. Open https://structured-content-2022-web-git-venue-names-show-main-venue.sanity.build/
2. Scroll down below the nav block and verify that the venues section looks like indicated in the Figma sketch

### CODE REVIEW

**Comment by oyvind-stenhaug** [comment]
Added my changes and marked the PR ready for review since I assume that's the only part that was missing.

File: web/components/TextBlock/VenuesSection/VenuesSection.tsx [hunk 1]
```diff
@@ -24,7 +29,7 @@ export const VenuesSection = ({
   const sectionVenues = getCollectionForSelectionType(type, allVenues, venues);
   return sectionVenues.length === 0 ? null : (
     <GridWrapper>
-      <VenueNames venues={sectionVenues} />
+      <VenueNames venues={sectionVenues} {...otherProps} />
```
[the remark this answers is not in the document]
Comment (j0nas) [reply web/components/TextBlock/VenuesSection/VenuesSection.tsx]: Good catch! This stems from a time where `<VenueNames />` was used in multiple places. Good to fix these things as one spots them. 206266a

**Comment by j0nas** [comment]
> I guess this is outside the scope of this PR, but it would be nice if "venues" and "allVenues" could be merged in the query, so that we don't waste resources fetching "allVenues" in the cases where it's not actually used.

I filed #152 for this. :slightly_smiling_face:

**Comment by oyvind-stenhaug** [comment]
LGTM

**Pull request merged**